scss/_patterns_navigation.scss

```diff
@@ -77,12 +77,11 @@ $navigation-hover-opacity: 0.58;
 
     &__links {
       list-style: none;
-      margin: 0;
+      margin: -1px 0 0 0;
       padding: 0;
-      @media (max-width: $breakpoint-navigation-threshold) {
-        margin-top: -1px; // prevents bottom border of nav from moving 1px
-      }
+
       @media (min-width: $breakpoint-navigation-threshold) {
+        margin-top: 0; // prevents bottom border of nav from moving 1px
```

Comment:
Order the properties alphabetically. I think this causes build to fail because of linting errors.